client/blocks/subscription-length-picker/index.jsx

```diff
@@ -0,0 +1,193 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal Dependencies
+ */
+import { getPlansBySiteId, isRequestingSitePlans } from 'state/sites/plans/selectors';
+import { localize } from 'i18n-calypso';
+import formatCurrency from 'lib/format-currency';
+import { getProductsList, isProductsListFetching } from 'state/products-list/selectors';
+import { requestProductsList } from 'state/products-list/actions';
+import { requestPlans } from 'state/plans/actions';
+import { getPlanRawPrice, isRequestingPlans } from '../../state/plans/selectors';
+import { getCurrentUserCurrencyCode } from '../../state/current-user/selectors';
+import { getPlanDiscountedRawPrice } from '../../state/sites/plans/selectors';
```

Comment:
Please use the full path here. It make it easier to search for usages. These boil down to `state/sites/plans/selectors` anyway.